x-pack/platform/plugins/shared/onechat/public/application/components/agents/edit/tools_selection.tsx

```diff
@@ -46,9 +46,9 @@ export const ToolsSelection: React.FC<ToolsSelectionProps> = ({
 }) => {
   // Group tools by provider
   const toolsByProvider = useMemo(() => {
```

Comment:
nitpick: can we update this to not use provider and use type?

Reply:
Yeah, it was a long refactor, I was just lazy on Friday. Fixed it - https://github.com/elastic/kibana/pull/227452/commits/bbff84e8ce9a0228fa0e492e46e0179c62943e2a